common/src/main/java/org/apache/rocketmq/eventbridge/tools/ARNHelper.java

```diff
@@ -0,0 +1,63 @@
+ /*
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements.  See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License.  You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package org.apache.rocketmq.eventbridge.tools;
+
+/**
+ * @Author changfeng
+ * @Date 2022/4/25 11:22 上午
+ */
+public class ARNHelper {
+    public static final String EVENTBUS_ALL = "eventbus/*";
+    public static final String EVENTBUS_PATTERN = "eventbus/%s";
+    public static final String EVENTRULE_ALL = "eventbus/%s/rule/*";
```

Comment:
ARN is the concept from aliyun ,  is inappropriate to use here.

Reply:
The corresponding class has been deleted, and the related configuration is placed in HttpEventAPIImpl.